Lesson4HomeWork.playground/Lesson4HomeWork.playground/Contents.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+
+let firstTupple: (Int, String) = (0, "zero")
+let secondTupple = (2, "two")
+let thirdTupple = (3, "three")
+let fourthTupple = (4, "four")
+let fifthTupple = (5, "five")
+let sixthTupple = (6, "six")
+let seventhTupple = (7, "seven")
+let eigthsTupple = (8, "eigth")
+
+var arrayTupples = [firstTupple, secondTupple, thirdTupple, fourthTupple, fifthTupple, sixthTupple, seventhTupple, eigthsTupple]
+var powArrayTupples = arrayTupples
```

Comment:
What's the point to create this variable? You can just use `arrayTupples` inside the cycle